### PR TITLE
conf staging implemented

### PR DESCRIPTION
**Pull request by georgestam**
@vemv, PR from issue #6 - Just `staging` environment added for a new heroku app. 

(Codeship built will not pass as there is the another PR opened that fixes robucop)

### CODE REVIEW

File: config/initializers/global_functions.rb [hunk 1]
```diff
@@ -10,10 +10,18 @@ def production?
   Rails.env.production?
 end
 
+def staging?
+  Rails.env.production?
```
Comment (vemv) [inline config/initializers/global_functions.rb]: wtf? that's one deceiving name if I ever saw one :)

production != staging, they're different environments

File: config/environments/staging.rb [hunk 2]
```diff
@@ -0,0 +1,89 @@
+Rails.application.configure do
```
Comment (vemv) [inline config/environments/staging.rb]: Regading configuration, it is key that staging remains as close to production as possible, now and in the future.

Therefore copy the production_or_staging.rb file/pattern from kuina.

File: config/initializers/global_functions.rb [hunk 3]
```diff
@@ -10,10 +10,18 @@ def production?
...
+
 def development_or_test?
   development? || test?
 end
 
 def development_or_production?
   development? || production?
+end
+
+def staging_or_production?
```
Comment (vemv) [inline config/initializers/global_functions.rb]: call it `production_or_staging?` - you should care/think about the most important thing first.

**Review by vemv — CHANGES_REQUESTED** [review]

**Comment by georgestam** [comment]
@vemv I have implemented the changes (codeship still fails due to rubocop fixed in another PR)

File: config/initializers/global_functions.rb [hunk 4]
```diff
@@ -10,10 +10,18 @@ def production?
...
 
+def staging?
+  Rails.env.staging?
+end
+
 def development_or_test?
   development? || test?
 end
 
 def development_or_production?
```
Comment (vemv) [inline config/initializers/global_functions.rb]: this is bit of a weird function to have, to you need it?

**Review by vemv — APPROVED** [review]
Approved, just possibly delete development_or_production (because it implies staging would return false)

File: config/initializers/global_functions.rb (hunk 4 above)
Comment (georgestam) [reply config/initializers/global_functions.rb]: I would like to test MailChimp subscription and Postmark on both, but not on development and testing. is this a normal procedure?

**Pull request merged**